index.html

```diff
@@ -928,6 +929,23 @@ <h4>Update Status</h4>
           data-api-endpoint="post /credentials/status"></div>
       </section>
 
+      <section>
+        <h4>Delete a Specific Credential</h4>
+        <p>
+An [=issuer service=] or a [=holder service=] might store an issued
+[=verifiable credential=] for an extended period of time. When this is done,
+it can be useful to delete such a [=verifiable credential=]. An [=issuer=]
+might need to do so because of regulatory requirements such as
+<a href="https://en.wikipedia.org/wiki/Right_to_be_forgotten">
```

Comment:
```suggestion
it can be useful to delete such a [=verifiable credential=]; for instance,
an [=issuer=] might need to do so because of regulatory requirements such
as <a href="https://en.wikipedia.org/wiki/Right_to_be_forgotten">
```